specifyweb/frontend/js_src/lib/tests/testmain.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+(cd specifyweb/frontend/js_src && npm run test)
```

Comment:
Does this shell script get used anywhere?

Reply:
I defined a local hook here:
https://github.com/specify/specify7/blob/f25006e0b1b70f63ecbc7984699f80a2d66a0027/.pre-commit-config.yaml#L40

All it needs is to run `npm run test` if any js or ts file was modified.

Since `npm` is a global command, rather than a local shell file, I wasn't able to get it to work nicely. So I created this local shell script that get's called by the hook.

Though, maybe there is a better way to do this.